src/http/RequestMethod.cc

```diff
@@ -145,7 +145,7 @@ HttpRequestMethod::isHttpSafe() const
     // RFC 5789 - none
     // RFC 5842 - none
 
-    // RFC 7540 section 11.6
+    // RFC 9113 section 3.4
```

Comment:
The old [change request](https://github.com/squid-cache/squid/pull/1071#discussion_r892709662) about this change was resolved without a comment, but the current code reflects no changes. Could be an earlier change lost in a forced push. In that case, here it is again:

```suggestion
    // // RFC 7540 section 11.6
```

Reply:
Per the old change request:

> I do not insist on any changes here,